src/RemoteInboxNotifications/README.md

```diff
@@ -497,3 +497,17 @@ WooCommerce Admin plugin.
 ```
 
 No other values are needed.
+
+## Debugging Specification
+
+You can see the evaluation of each specification by turning on an optional evaluation logger.
+
+1. Define `WC_ADMIN_DEBUG_RULE_EVALUATOR` constant in `wp-config.php`. `define('WC_ADMIN_DEBUG_RULE_EVALUATOR', true);`
+2. Run `wc_admin_daily`
+3. cd to `wp-content/uploads/wc-logs/` and check a log file in `remote-inbox-notifications-:date-hash.log` format.
```

Comment:
Just a heads up that this may be tricky for Globalstep to test.  Not sure if testing instructions are warranted here or not.

Reply:
That's a good point 🤔 I'll try to see if there's a simpler way to test this logger for GlobalStep.

Reply:
@joshuatf I don't think we want GlobalStep to test this feature as this logger is intended for dev use only.